qa/L0_perf_pyclients/test.sh

```diff
@@ -43,8 +43,10 @@ REPORTER=../common/reporter.py
 CLIENT_LOG="./simple_perf_client.log"
 SIMPLE_PERF_CLIENT=simple_perf_client.py
 
+TF_VERSION=${TF_VERSION:=2}
```

Comment:
It is good for default, but let's still provide `TF_VERSION=1` for `L0_perf_*` from the gitlab yml script.

Reply:
Yes. `TF_VERSION=1` is added for `L0_perf_*` test in the gitlab yml file.